daq_macros.py

```diff
@@ -3340,35 +3340,35 @@ def setProcRam():
   else:
     db_lib.setBeamlineConfigParam("fmx","spotNode1","xf17id2-ws6")
     db_lib.setBeamlineConfigParam("fmx","spotNode2","xf17id2-ws6")
-    db_lib.setBeamlineConfigParam("fmx","cbfComm","/usr/local/MX-Soft/bin/eiger2cbfJohn")
-    db_lib.setBeamlineConfigParam("fmx","dialsComm","/usr/local/MX-Soft/Phenix/phenix-installer-dev-2666-intel-linux-2.6-x86_64-centos6/build/bin/dials.find_spots_client")        
+    db_lib.setBeamlineConfigParam("fmx","cbfComm","eiger2cbf")
+    db_lib.setBeamlineConfigParam("fmx","dialsComm","dials.find_spots_client")        
     
 
 def setProcGPFS():
   if (daq_utils.beamline == "amx"):
-    db_lib.setBeamlineConfigParam("amx","spotNode1","cpu-002")
-    db_lib.setBeamlineConfigParam("amx","spotNode2","cpu-003")
-    db_lib.setBeamlineConfigParam("amx","spotNode3","cpu-002")
-    db_lib.setBeamlineConfigParam("amx","spotNode4","cpu-003")
-    db_lib.setBeamlineConfigParam("amx","spotNode5","cpu-002")
-    db_lib.setBeamlineConfigParam("amx","spotNode6","cpu-003")    
-    db_lib.setBeamlineConfigParam("amx","cbfComm","/usr/local/crys-local/bin/eiger2cbf-linux")
-    db_lib.setBeamlineConfigParam("amx","dialsComm","/usr/local/crys-local/phenix-1.12-2829/build/bin/dials.find_spots_client")        
+    db_lib.setBeamlineConfigParam("amx","spotNode1","uranus-cpu002")
+    db_lib.setBeamlineConfigParam("amx","spotNode2","uranus-cpu003")
+    db_lib.setBeamlineConfigParam("amx","spotNode3","uranus-cpu002")
+    db_lib.setBeamlineConfigParam("amx","spotNode4","uranus-cpu003")
+    db_lib.setBeamlineConfigParam("amx","spotNode5","uranus-cpu002")
+    db_lib.setBeamlineConfigParam("amx","spotNode6","uranus-cpu003")    
+    db_lib.setBeamlineConfigParam("amx","cbfComm","eiger2cbf")
+    db_lib.setBeamlineConfigParam("amx","dialsComm","dials.find_spots_client")
     
   else:
-    db_lib.setBeamlineConfigParam("fmx","spotNode1","cpu-007")
-    db_lib.setBeamlineConfigParam("fmx","spotNode2","cpu-008")
-    db_lib.setBeamlineConfigParam("fmx","spotNode3","cpu-009")
-    db_lib.setBeamlineConfigParam("fmx","spotNode4","cpu-010")
-    db_lib.setBeamlineConfigParam("fmx","spotNode5","cpu-008")
-    db_lib.setBeamlineConfigParam("fmx","spotNode6","cpu-004")        
-    db_lib.setBeamlineConfigParam("fmx","cbfComm","/usr/local/crys-local/bin/eiger2cbf-linux")
-    db_lib.setBeamlineConfigParam("fmx","dialsComm","/usr/local/crys-prod/phenix-1.11rc2-2531/build/bin/dials.find_spots_client")
+    db_lib.setBeamlineConfigParam("fmx","spotNode1","uranus-cpu007")
+    db_lib.setBeamlineConfigParam("fmx","spotNode2","uranus-cpu008")
+    db_lib.setBeamlineConfigParam("fmx","spotNode3","uranus-cpu009")
+    db_lib.setBeamlineConfigParam("fmx","spotNode4","uranus-cpu010")
+    db_lib.setBeamlineConfigParam("fmx","spotNode5","uranus-cpu008")
+    db_lib.setBeamlineConfigParam("fmx","spotNode6","uranus-cpu004")        
+    db_lib.setBeamlineConfigParam("amx","cbfComm","eiger2cbf")
```

Comment:
Should these two be amx?

Reply:
this is one of the functions that seems to be used for potentially re-setting up the config parameters, such as for a new beamline, or as examples of what to put in for a new beamline. I have merely replaced the hostnames with their new equivalents by adding "uranus-". there are amx equivalents for spotNodes a few lines above, lines 3354-3360.